specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2022-10-01/Workspaces.json

```diff
@@ -0,0 +1,735 @@
+          "description": "Workspace creation date.",
+          "readOnly": true
+        },
+        "modifiedDate": {
+          "type": "string",
+          "description": "Workspace modification date.",
+          "readOnly": true
+        },
+        "publicNetworkAccessForIngestion": {
+          "description": "The network access type for accessing Log Analytics ingestion.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "publicNetworkAccessForQuery": {
+          "description": "The network access type for accessing Log Analytics query.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "forceCmkForQuery": {
+          "type": "boolean",
+          "description": "Indicates whether customer managed storage is mandatory for query management."
+        },
+        "privateLinkScopedResources": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkScopedResource"
+          },
+          "x-ms-identifiers": [
+            "resourceId"
+          ],
+          "description": "List of linked private link scope resources."
+        },
+        "features": {
+          "$ref": "#/definitions/WorkspaceFeatures",
+          "description": "Workspace features."
+        },
+        "defaultDataCollectionRuleResourceId": {
+          "type": "string",
+          "description": "The resource ID of the default Data Collection Rule to use for this workspace. Expected format is - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/dataCollectionRules/{dcrName}."
+        }
+      },
+      "description": "Workspace properties."
+    },
+    "WorkspaceFeatures": {
+      "properties": {
+        "enableDataExport": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Flag that indicate if data should be exported."
+        },
+        "immediatePurgeDataOn30Days": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Flag that describes if we want to remove the data after 30 days."
+        },
+        "enableLogAccessUsingOnlyResourcePermissions": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Flag that indicate which permission to use - resource or workspace or both."
+        },
+        "clusterResourceId": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "Dedicated LA cluster resourceId that is linked to the workspaces."
+        },
+        "disableLocalAuth": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Disable Non-AAD based Auth."
+        }
+      },
+      "additionalProperties": true,
+      "description": "Workspace features."
+    },
+    "Identity": {
+      "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json",
```

Comment:
this is not valid usage, please remove this definition, and refer to ""../../../../../common-types/resource-management/v3/managedidentity.json" in the properties directly